client/__stories__/pages/signUpIndex.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SearchInput from '@/components/SearchInput';
+import { ComponentStory } from '@storybook/react';
+import Index from '@/pages/signUp';
+
+export default {
+  component: Index,
+  title: 'signUpIndex',
```

Comment:
```suggestion
  title: 'page/SignUpPage',
```
정도로 바꿔서 컴포넌트들과 구분짓는 건 어떨까요?

Reply:
오 이런 방법이 있군요.